cli/cmd/vulnerability.go

```diff
@@ -371,6 +399,34 @@ func pollScanStatus(requestID string) error {
 	}
 }
 
+func vulnFailureThreshold(assessment api.HostVulnHostAssessment) {
+	severityRating, _ := eventSeverityToProperTypes(assessment.HighestSeverity())
+	fixableSeverityRating, _ := eventSeverityToProperTypes(assessment.HighestFixableSeverity())
+	threshold, _ := eventSeverityToProperTypes(vulCmdState.FailOnSeverity)
+
+	if vulCmdState.FailOnFixable && severityRating <= fixableSeverityRating {
+		os.Exit(1)
+	}
+
+	if !vulCmdState.FailOnFixable && severityRating <= threshold {
+		os.Exit(1)
+	}
+}
+
+func vulnContainerFailureThreshold(assessment *api.VulnContainerAssessment) {
+	severityRating, _ := eventSeverityToProperTypes(assessment.HighestSeverity())
```

Comment:
Move to vulnerabilities_container.go as` func (assessment **VulnContainerAssessment) vulnFailureThreshold()`